lib/tlm/tlm_interpreter.go

```diff
@@ -9,11 +9,12 @@ import (
 
 // Measurement represents a single measurement in a telemetry packet.
 type Measurement struct {
-	Name       string `yaml:"name"`                 // Name of the measurement
-	Size       int    `yaml:"size"`                 // Size of the measurement in bytes
-	Type       string `yaml:"type,omitempty"`       // Type of the measurement (int, float)
-	Unsigned   bool   `yaml:"unsigned,omitempty"`   // Whether the measurement is unsigned
-	Endianness string `yaml:"endianness,omitempty"` // Endianness of the measurement (big, little)
+	Name       string  `yaml:"name"`                 // Name of the measurement
+	Size       int     `yaml:"size"`                 // Size of the measurement in bytes
+	Type       string  `yaml:"type,omitempty"`       // Type of the measurement (int, float)
+	Unsigned   bool    `yaml:"unsigned,omitempty"`   // Whether the measurement is unsigned
+	Endianness string  `yaml:"endianness,omitempty"` // Endianness of the measurement (big, little)
+	Scaling    float64 `yaml:"scaling,omitempty"`    // Scaling factor for the measurement (optional)
```

Comment:
Might want to add a comment (here and/or in FSW) that the scaling factor is multiplied here and divided by in FSW. I was trying to think of a more self explanatory name that implies divide on send, multiply on receive but i couldnt think of anything but scaling seems like it could go either way.

Reply:
Although, i suppose the scaling happens in code in FSW rather than in Config like it does here so maybe it doesnt matter that much. either way, it should probably be documented in a yaml comment or a go comment somewhere

Reply:
Im just gonna rename Scaling to say ScalingFactor. Factor should signify multiplication, not division